### ISSUE TITLE
Not compatible with rollup v4

### ISSUE THREAD

**Issue by jamesgpearce** [mjeanroy/rollup-plugin-prettier#1394]
```
npm WARN Could not resolve dependency:
npm WARN peer rollup@"^1.0.0 || ^2.0.0 || ^3.0.0" from rollup-plugin-prettier@4.0.0
```

JFYI!

**Comment by mjeanroy** [comment]
Thanks, it's now fixed with version `v4.1.0` (published a few minutes ago).

**Comment by jamesgpearce** [comment]
thank you! - awesome
